### PR TITLE
Feature/ingredients adjustments

### PR DESCRIPTION
**Pull request by lbebber**
Adds auto scroll to the next step ("ingredients") when the user selects a variation.

Also fixes a visual bug when the Add to Cart button was loading.

Also fixes a visual bug when selecting variations, which was changing the height of the selected items.

### CODE REVIEW

File: react/components/Buttons/AddToCart.js [hunk 1]
```diff
@@ -32,13 +32,15 @@ class AddToCart extends Component {
...
       <div className={`actions--add-to-cart tc pa5 ${isModalOpen ? 'w-100 left-0 bg-white z-4 bottom-0 bt b--light-gray' : ''}`}>
         <Button type="submit" onClick={onSubmit} isLoading={isLoading} block>
-          <FormattedMessage id="product-customizer.add-to-cart" />
-          <ProductPrice
-            showLabels={false}
-            showListPrice={false}
-            sellingPrice={total}
-            listPrice={total}
-          />
+          <div className="flex flex-grow-1 justify-between items-center">
```
Comment (kevinch) [inline react/components/Buttons/AddToCart.js]: `flex-grow` is normally a child property of a parent with `flex`. Does it make sense here?

File: react/components/ProductCustomizer.js [hunk 2]
```diff
@@ -177,7 +187,8 @@ class ProductCustomizer extends Component {
    */
   handleVariationChange = async variationObject => {
     const { productQuery: { product } } = this.props
-    const variationSku = variation && variationObject.skuId
+    const variationSku = variationObject && variationObject.skuId
+
     const sku = product.items.find(sku => sku.itemId === variationSku)
 
     // TODO: add proper error message to handle null variationObject and sku
```
Comment (kevinch) [inline react/components/ProductCustomizer.js]: This would be better as an issue.

File: react/components/ProductCustomizer.js [hunk 3]
```diff
@@ -414,6 +443,12 @@ class ProductCustomizer extends Component {
               skus={requiredVariations}
               onVariationChange={this.handleVariationChange}
             />
+            <div ref={this.ingredientsContentAnchor}
+              style={{
+                position: "relative",
```
Comment (klzns) [inline react/components/ProductCustomizer.js]: ⛏ Podia usar `className="relative"`

File: react/components/ProductCustomizer.js (hunk 2 above)
Comment (lbebber) [reply react/components/ProductCustomizer.js]: Created the issue: https://github.com/vtex-apps/product-customizer/issues/7

Will leave the todo here as well though, to find the code more easily later.

File: react/components/Buttons/AddToCart.js (hunk 1 above)
Comment (lbebber) [reply react/components/Buttons/AddToCart.js]: I thought the parent was `flex` too, but I was wrong. Setting it to `w-100`

**Pull request merged**